industrial_robot:
    diff_drive_controller:
       type: "diff_drive_controller/DiffDriveController"
       wheel_separation: 0.53 # actually 0.53 but orientation is for unknown reasons better with this one
       wheel_radius: 0.164
       publish_rate: 50.0

       wheel_separation_multiplier: 1.0
       wheel_radius_multiplier: 1.0

       left_wheel: 'wheel_support_left_to_wheel_left'
       right_wheel: 'wheel_support_right_to_wheel_right'

       # Odometry covariances for the encoder output of the robot. These values should
       # be tuned to your robot's sample odometry data, but these values are a good place
       # to start
       pose_covariance_diagonal: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0] #[0.001, 0.001, 0.001, 0.001, 0.001, 0.003]
       twist_covariance_diagonal: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0] #[0.001, 0.001, 0.001, 0.001, 0.001, 0.003]

       # Top level frame (link) of the robot description
       base_frame_id: base_link

       # Velocity and acceleration limits for the robot
       linear:
         x:
	   has_velocity_limits    : true
	   max_velocity           : 3.0  # m/s
	   has_acceleration_limits: true
	   max_acceleration       : 6.0   # m/s^2
       angular:
         z:
           has_velocity_limits    : true
	   max_velocity           : 3.0   # rad/s
           has_acceleration_limits: true
	   max_acceleration       : 6.0   # rad/s^2
